src/docs/tool-libfuzzer.dox

```diff
@@ -20,27 +20,27 @@ Compiling with Clang's Address Sanitizer isn't mandatory but it is recommended s
 exposes memory bugs.
 
 @code
-$ cd build_posix/
-$ ../configure --enable-libfuzzer CC="clang-8" CFLAGS="-fsanitize=address"
+$ mkdir build
+$ cmake -DENABLE_LLVM=1 -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/clang.cmake -DCLANG_C_VERSION="8" -DCLANG_CXX_VERSION="8" -DCMAKE_BUILD_TYPE=ASan ../.
 @endcode
 
 ## Step 2: Build as usual
 
 @code
-$ cd build_posix
+$ cd build
```

Comment:
Should this line be moved up after line 23?

Reply:
Yep, good find. Fixed 5d464c9